api/server/router/image/backend.go

```diff
@@ -32,9 +32,9 @@ type imageBackend interface {
 }
 
 type importExportBackend interface {
-	LoadImage(ctx context.Context, inTar io.ReadCloser, outStream io.Writer, quiet bool) error
+	LoadImage(ctx context.Context, inTar io.ReadCloser, platform *ocispec.Platform, outStream io.Writer, quiet bool) error
 	ImportImage(ctx context.Context, ref reference.Named, platform *ocispec.Platform, msg string, layerReader io.Reader, changes []string) (dockerimage.ID, error)
-	ExportImage(ctx context.Context, names []string, outStream io.Writer) error
+	ExportImage(ctx context.Context, names []string, platform *ocispec.Platform, outStream io.Writer) error
```

Comment:
Starting to wonder if we should (also) have a options-type defined for these endpoints, as the list of arguments is starting to grow.

Reply:
I think it's still fine, as it's only one parameter that is a truly "option" (inTar and outStream aren't really "options").
It's also only an internal interface, so we don't need to worry about breaking compatibility in future.
I don't think we're gonna add another parameter any soon, and if we do, then we can extract it to a struct.